website/content/docs/concepts/acl/auth-methods/oidc.mdx

```diff
@@ -74,6 +74,107 @@ Complete the login via your OIDC provider. Launching browser to:
 Your browser opens to the generated URL to complete the provider's login.
 Enter the URL manually if the browser does not automatically open.
 
+### Client assertions
+
+Also known as "[private key JWT][]", client assertions offer a more secure
+authentication mechanism compared to client secrets. The OIDC client (in this
+case, Nomad) signs a JWT with an RSA private key (or HMAC key) that the OIDC
+provider verifies with an associated public key (or the same HMAC key).
```

Comment:
```suggestion
authentication mechanism compared to client secrets. The OIDC client, which in this
case is Nomad, signs a JWT with an RSA private key, or HMAC key, that the OIDC
provider verifies with an associated public key, or the same HMAC key.
```

style: moved important into out of parentheses

Reply:
I want to push back on this one a bit, because I think it's harder to follow the main thrust across the extra commas. In particular, I would be okay with some people skating past "(or HMAC key)" because it's the less secure option, and I think people who care about it will be looking for it specifically.